package.json

```diff
@@ -42,6 +42,6 @@
     "eslint": "^5.6.0",
     "get-port": "^4.0.0",
     "got": "^9.2.2",
-    "nodemon": "^1.18.4"
+    "nodemon": "^1.19.1"
```

Comment:
Had to update this since I had an issue with `event-stream@3.3.6` doesn't look like it exists anymore 🤔.